cli/ballerina-packerina/src/main/java/org/ballerinalang/packerina/cmd/RunCommand.java

```diff
@@ -93,7 +93,7 @@
     private boolean experimentalFlag;
 
     public RunCommand() {
-        this.outStream = System.out;
+        this.outStream = System.err;
         this.errStream = System.err;
```

Comment:
Shall we clean this up by removing the duplicate references to the same stream? Also, in other commands as well, such as `build`?

Reply:
It will be somewhat of a work as there are multiple references in the packerina side. Shall we push this to next milestone ?

Reply:
+1. Can you create an issue to track that?

Reply:
https://github.com/ballerina-platform/ballerina-lang/issues/19206